helm/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -46,6 +46,9 @@ spec:
       {{- if .Values.hostNetwork }}
       hostNetwork: true
       {{- end }}
+      {{- with .Values.dnsPolicy }}
```

Comment:
i think it's better to use if to avoid renderDNSPolicy at all if dnsPolicy is not set
```
{{- if .Values. dnsPolicy }}
dnsPolicy: {{ .Values.dnsPolicy }}
{{- end }}
```

Reply:
fixed.